agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -259,9 +259,7 @@ async def fetch_base_by_id(base_id: str) -> Optional[VariantBaseDB]:
     """
     if base_id is None:
         raise Exception("No base_id provided")
-    base = await VariantBaseDB.find_one(
-        VariantBaseDB.id == ObjectId(base_id), fetch_links=True
-    )
+    base = await VariantBaseDB.find_one(VariantBaseDB.id == ObjectId(base_id))
```

Comment:
are we sure we are not breaking anything

Reply:
This breaks in commons [this](https://github.com/Agenta-AI/agenta-commons/blob/6666f5ecb36012431eb84aa184a6a8046bcfd60a/agenta-backend/agenta_backend/commons/services/db_manager_ee.py#L749
)
Which breaks [this](https://github.com/Agenta-AI/agenta-commons/blob/6666f5ecb36012431eb84aa184a6a8046bcfd60a/agenta-backend/agenta_backend/commons/utils/permissions.py#L179)

Reply:
[this](https://github.com/Agenta-AI/agenta/blob/3180c676164a4d099851c8f76c8c33a707cc3f65/agenta-backend/agenta_backend/routers/configs_router.py#L114) too no? Or do we have access to the id without having to fetch?

Reply:
[this](https://github.com/Agenta-AI/agenta/blob/3180c676164a4d099851c8f76c8c33a707cc3f65/agenta-backend/agenta_backend/routers/container_router.py#L184) too no?

Reply:
> [this](https://github.com/Agenta-AI/agenta/blob/3180c676164a4d099851c8f76c8c33a707cc3f65/agenta-backend/agenta_backend/routers/container_router.py#L184) too no?

Nope. The deployment field type in `VariantBaseDB` is of `PydanticObjectId`.

Reply:
Did it turn out that way?

Reply:
Sadly yes, I brought it up earlier today.

Reply:
Can it be fixed by using something like ObjectId(variant.base) or something like that. Because at the end of the day variant.base = ObjectId(variant.base.id)

Reply:
It looks like Link has a property `ref` which returns a DBREf https://github.com/BeanieODM/beanie/blob/51e73ebc297df02082c5c683d555f7d057c4f6aa/beanie/odm/fields.py#L287
And the DBRef has a property id
https://pymongo.readthedocs.io/en/stable/api/bson/dbref.html#bson.dbref.DBRef

Reply:
Yup, this works. Thank you.

Reply:
This works, thank you.